Wordle.Api/Wordle.Api/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Models;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class PlayerController(PlayerService playerService) : ControllerBase
+{
+    
+    [HttpGet("Player")]
+    public async Task<Player?> GetPlayer(string playerName = "Guest")
+    {
+        return await playerService.GetPlayer(playerName) ?? null;
```

Comment:
I am a little confused by what you are using this player for, the only player I saw in the assignment was the database itself that the info for the leaderBoard was saved in

Reply:
It was not finished. Thanks for suggestion